packages/ui/src/components/editor/properties/audio/analysis/index.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/EtherealEngine/etherealengine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Ethereal Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Ethereal Engine team.
+
+All portions of the code written by the Ethereal Engine team are Copyright © 2021-2023 
+Ethereal Engine. All Rights Reserved.
+*/
+
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { useComponent } from '@etherealengine/ecs'
+import {
+  EditorComponentType,
+  commitProperty,
+  updateProperty
+} from '@etherealengine/editor/src/components/properties/Util'
+import { AudioAnalysisComponent } from '@etherealengine/engine/src/scene/components/AudioAnalysisComponent'
+import Slider from '../../../../../primitives/tailwind/Slider'
+import BooleanInput from '../../../input/Boolean'
+import InputGroup from '../../../input/Group'
+import PropertyGroup from '../../group'
+
+export const AudioAnalysisEditor: EditorComponentType = (props) => {
+  const { t } = useTranslation()
+  const audioAnalysisComponent = useComponent(props.entity, AudioAnalysisComponent)
+  return (
```

Comment:
no way to delete this editor for some reason.

Reply:
its using a property group with no close functionality
i have replaced it with a node editor
broadly if we have components we dont want to be deleted from the scene, we can use propertyGroup wthout an onClose

and use Nodeeditor in general